frontend/plugins/tourism_ui/src/modules/pms/types/branch.ts

```
@@ -0,0 +1,51 @@
+export interface IPmsPaymentType {
+  _id: string;
+  type: string;
+  title: string;
+  icon?: string;
+  config?: string;
+}
+
+export interface IPmsDiscount extends IPmsPaymentType {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)